README.md

```diff
@@ -4,6 +4,10 @@ A simple command line utility to tell you if your lightmaps are too high resolut
 
 ## Usage
 
+The program can be run with `LightmapUtil.exe filename.bsp`. This generates `filename-LightmapUtilLog.txt` that provides you with details about lightmapping on that map file.
```

Comment:
you should add the linux equivalent here `lightmaputil filename.bsp` 😀

Reply:
you don't even need the .exe on windows, but its really not a big deal rn, linux builds aren't provided (yet)